WcaOnRails/app/views/competitions/my_competitions.html.erb

```diff
@@ -3,22 +3,28 @@
 <div class="container">
   <h2><%= yield(:title) %></h2>
 
-  <%= wca_table_for @competitions, extra_table_class: "competitions-table", hover: false, striped: false do |table| %>
-    <% table.column :name %>
-    <% table.column :location do |competition| %>
-      <%= competition.cityName %>, <%= competition.country_name %>
-    <% end %>
-    <% table.column :start_date %>
-    <% table.column :delegates %>
-    <% table.column :organizers %>
-    <% table.column do |competition| %>
-      <% if current_user.can_manage_competition?(competition) %>
-        <%= link_to "Edit", edit_competition_path(competition) %>
+  <% if @competitions.length < 1 %>
+    <div class="alert alert-info" role="alert">
+      You currently have no competitions! Check out the <%= link_to "competitions list", "results/competitions.php" %>.
```

Comment:
Hmmm... does this link actually work? Lets always use absolute paths, so `/results/competitions.php`.